roadmap.md

```diff
@@ -0,0 +1,43 @@
+# Usage of Epics
+
+An Epic is a grouping of smaller related issues/tasks that fits into a larger goal. An Epic can potentially span across multiple iterations. 
+
+To track how smaller issues fit into the larger goal, we use [task lists](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-task-lists).
+Breaking a large issue into smaller issues makes the work more manageable and enables team members to work in parallel. 
+It also leads to smaller pull requests, which are easier to review.
+
+>NOTE: To create an epic on Github, use the available Epic template.
+
+# Usage of Github issue template/forms
+
+We have [customized templates](https://github.com/bf2fc6cc711aee1a0c2a/ffm-project/blob/main/.github/ISSUE_TEMPLATE/epic.md) that helps contributors in 
+creating new issues in the repository. When creating an issue, choose the template that best fits the issue you are about to create.  
+
+# Communicate
+- Use `@mentions` to alert a person or entire team about a the issue. 
+- Use appropriate labels during the creation or the triaging of the issue. Labels help to categorise issues thus improving their discoverbility. Check out the existing labels on - https://github.com/bf2fc6cc711aee1a0c2a/ffm-project/labels.  
+- Assign collaborators to issues to communicate responsibility.
+
+>NOTE: You can always reach out to the wider team via the available [communication channels](https://github.com/bf2fc6cc711aee1a0c2a/ffm-project/wiki/How-to-communicate-in-the-project) 
+
+# View the Roadmap
+
+The roadmap is defined in the [project view](https://github.com/orgs/bf2fc6cc711aee1a0c2a/projects/5) 
+We have two main views:
+- The roject view containing all the issues that have been triaged and potentially assigned to a Sprint.
+- The Epic view, containing larger goals/tasks.
+
+The two views are the source of truth and they are kept up to date to showing our roadmap.
+
+## How to add an issue to the Roadmap (TODO)
+
+## How to add an Epic to the Roadmap (TODO)
+
```

Comment:
I have captured David thoughts on having two views:
- the project view containing all issues assigned to an iteration. I think some issues (un-triaged ones) might be assigned to an iteration and that is fine as we can consider those issues to be the one in the backlog
- the epic view containing larger chunks of work 

This is reasonable enough as the epic view will complement the project view containing all issues. Some people might be interested in just seeing the big picture and its ETA.
The project view will contain all issues, useful enough to show the ETA of individual issues. I can also see Bugs report issue to be seen here as they do not have a parent Epic.

Reply:
That makes sense to me @machi1990 . So the project view is ultimately the sprint board and backlog combined? The Epic view is looking at the higher level scope of work.

Reply:
> So the project view is ultimately the sprint board and backlog combined? 

Correct. And if someone is interested on seeing the work on a given sprint e.g current sprint, filters can be leveraged. This might be simplistic but it avoid the case of juggling through and maintaining  many views.

Reply:
> I have captured David thoughts on having two views:
> 
> * the project view containing all issues assigned to an iteration. I think some issues (un-triaged ones) might be assigned to an iteration and that is fine as we can consider those issues to be the one in the backlog
> * the epic view containing larger chunks of work
> 
> This is reasonable enough as the epic view will complement the project view containing all issues. Some people might be interested in just seeing the big picture and its ETA. The project view will contain all issues, useful enough to show the ETA of individual issues. I can also see Bugs report issue to be seen here as they do not have a parent Epic.

As I continue to tweak the board, I feel like we can have only one project view  containing all the issues.
Then the epic view can simply simply be obtained using the `label: kind/epic` filter. 
The fundamental reason is that, if we add all the issues to the project view (epics, issues under epic and bugs) this also ends up messing the epic view because everything will be contained there too. So one has to filter out to get the epics.

Reply:
It is not possible to force a filter for an individual view?

Reply:
Unfortunately, I could not find a way to do so.